model/src/main/kotlin/config/PackageConfiguration.kt

```diff
@@ -93,6 +93,10 @@ data class PackageConfiguration(
             return false
         }
 
+        if (sourceCodeOrigin == null && sourceArtifactUrl == null && vcs == null) {
+            return true
+        }
+
```

Comment:
Should we organize the code so that we keep on having one section for each of the `match` properties? Like this:

```
if (sourceArtifactUrl != null) {
    return provenance is ArtifactProvenance && provenance.sourceArtifact.url
}

if (vcs != null) {
    return provenance is RepositoryProvenance && vcs.matches(provenance)
}

return true
```